wikimedia/wikipedia-ios
# Watchlist - add metrics

**Phabricator:** 
https://phabricator.wikimedia.org/T334968

### Notes
This PR is dependent on https://gerrit.wikimedia.org/r/c/schemas/event/secondary/+/955405. You will see console errors otherwise.

### Test Steps
1. Just code review on this and confirm that a couple of actions submit successfully to the backend. We can lean on QA for more extensive testing.
